posix.mak

```diff
@@ -353,6 +353,11 @@ $(DMD) : ${DMD_DIR}
 $(DMD_REL) : ${DMD_DIR}-${LATEST}
 	${MAKE} --directory=${DMD_DIR}-${LATEST}/src -f posix.mak AUTO_BOOTSTRAP=1 -j 4
 
+dmd-release : $(STD_DDOC) $(DMD_DIR) $(DMD)
+	$(MAKE) AUTO_BOOTSTRAP=1 --directory=$(DMD_DIR) -f posix.mak html \
+		DOCDIR=${DOC_OUTPUT_DIR}/dmd \
+		DOCFMT="$(addprefix `pwd`/, $(STD_DDOC))"
```

Comment:
The HTML for ddoc will get built from now on, but without a menu entry it's not visible to the user.